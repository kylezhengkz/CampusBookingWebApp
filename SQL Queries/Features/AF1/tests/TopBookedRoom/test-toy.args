{
    "args": [
        "00000000-0000-0000-0000-000000000001",
        null,
        null,
        1
    ],
    "kwargs": {}
}   