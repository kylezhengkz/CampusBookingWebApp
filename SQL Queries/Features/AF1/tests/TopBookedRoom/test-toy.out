{'roomID': UUID('00000000-0000-0000-0000-000000000001'), 'roomName': 'MC 4059', 'bookingCount': 1}